Benchling.js

```diff
@@ -0,0 +1,171 @@
+
+As with any scraper, this might break over time if Benchling changes their HTML structure.
+If this happens, feel free to reach out to me on the Benchling Community Forum
+https://community.benchling.com
+*/
+
+function detectWeb(_doc, _url) {
+	// Always return document since the target already restricts to ELN entries
+	return "document";
+}
+
+function doWeb(doc, url) {
+	scrape(doc, url);
+}
+
+function scrape(doc, url) {
+	var item = new Zotero.Item('document');
+	
+	// Get title from page title tag as fallback
+	var pageTitle = doc.querySelector('title');
+	var shortTitle = pageTitle ? pageTitle.textContent.split('·')[0].trim() : 'Untitled Benchling ELN Entry';
+	
+	// Find the title from the TitleEditor
+	var titleEditable = doc.querySelector('.mediocre-titleEditor-titleEditable');
+	if (titleEditable) {
+		shortTitle = titleEditable.textContent.trim();
+	}
+	item.shortTitle = shortTitle;
+	
+	// Find the Display experiment ID from the human-id span
+	var expId = '';
+	var humanId = doc.querySelector('#human-id');
+	if (humanId) {
+		expId = humanId.textContent.trim();
+	}
+	
+	// Fallback: If not found, maybe the metadata tab is active and the ID is in a different location.
+	if (!expId) {
+		var metadataIdSection = doc.querySelector('[data-test-element="humanIdSection"]');
+		if (metadataIdSection) {
+			// Get the text content after the "ID" header
+			var sectionText = metadataIdSection.textContent.trim();
+			// Remove "ID" from the beginning if present
+			expId = sectionText.replace(/^ID\s*/, '').trim();
+		}
+	}
+	
+	// If we found an EXP ID, add it to the title
+	if (expId) {
+		item.title = shortTitle + ' [' + expId + ']';
+	}
+	else {
+		item.title = shortTitle;
+	}
+	
+	// Find authors from metadata field. Only works if the Metadata tab is visible.
+	var authorField = doc.querySelector('.metadata-field--authorList');
+	if (authorField) {
+		var authorLinks = authorField.querySelectorAll('a[href]');
+		for (var i = 0; i < authorLinks.length; i++) {
+			var authorName = authorLinks[i].textContent.trim();
+			if (authorName) {
+				// Try to parse the name into first/last
+				var lastSpaceIndex = authorName.lastIndexOf(' ');
+				if (lastSpaceIndex > 0) {
+					// Has a space, assume "First Last"
+					item.creators.push({
+						firstName: authorName.substring(0, lastSpaceIndex),
+						lastName: authorName.substring(lastSpaceIndex + 1),
+						creatorType: 'author'
+					});
+				}
+				else {
+					// No space, single field
+					item.creators.push({
+						lastName: authorName,
+						creatorType: 'author',
+						fieldMode: 1
+					});
+				}
+			}
+		}
+	}
+	
+	// Find the review status badge. Only works if the Notes tab of the ELN entry is visible.
+	var reviewBadge = doc.querySelector('#badge-button');
+	var extraInfo = [];
+	if (reviewBadge) {
+		var reviewText = reviewBadge.textContent.trim();
+		if (reviewText) {
+			extraInfo.push('Review status: ' + reviewText);
```

Comment:
This is probably better as a note:


```suggestion
			item.notes.push({ note: 'Review status: ' + reviewText });
```